app/src/main/java/net/aiscope/gdd_app/ui/metadata/MetadataPresenter.kt

```diff
@@ -16,7 +16,8 @@ data class ViewStateModel(
     val disease: String,
     val images: List<File>,
     val options: List<FieldOption>,
-    val required: Boolean = true
+    val required: Boolean = true,
+    val sampleMetadata: SampleMetadata? = null
```

Comment:
Do you think that instead of adding the domain model as a field of the view state model, it would be better to add the individual fields that we need? Also, maybe we can do the mapping with the `MetadataMapper` here, instead of the activity, just a thought.

Reply:
Passing the individual fields makes sense. If we change to do the mapping with MetadataMapper to be in MetadataPresenter, does it make sense to move the reverse mapping (using MetadataMapper) that is done during save to also be in the MetadataPresenter rather than MetadataActivity?

Reply:
Yes, I would think so :)

Reply:
Thanks, I made this change in changeset c0f5c9692349070bbd81db509eab0ed21928085e.